Yubico.YubiKey/src/Yubico/YubiKey/Piv/Commands/DeleteKeyCommand.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2024 Yubico AB
+// 
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Yubico.Core.Iso7816;
+
+namespace Yubico.YubiKey.Piv.Commands
+{
+    /// <summary>
+    /// The <see cref="DeleteKeyCommand"/> is used to Delete a PIV key from the target slot
+    /// <remarks>
+    /// Any key, including the attestation key can be deleted.
+    /// </remarks>
+    /// </summary>
+    public class DeleteKeyCommand : IYubiKeyCommand<DeleteKeyResponse>
+    {
+        /// <summary>
+        /// The Yubikey slot of the key you want to delete.
+        /// </summary>
+        public byte SlotToClear { get; set; }
+
+        /// <summary>
+        /// Constructor for the <see cref="DeleteKeyCommand"/> which is used to delete a PIV key from a slot.
+        /// </summary>
+        /// <param name="slotToClear">The Yubikey slot of the key you want to clear.</param>
+        public DeleteKeyCommand(byte slotToClear)
```

Comment:
I know the SDK was never consistent about this, but I always liked including the default constructor as well. That way someone could using the object initialization syntax, which I always thought was better suited for the command objects.

```csharp
// Constructor
var cmd = new DeleteKeyCommand(0x9A);

// Object initialization (+ no var)
DeleteKeyCommand cmd = new()
{
    SlotToClear = 0x9A
};
```

For this specific command, the latter is definitely more verbose... but for other more complex commands it can end up overall being a bit nicer.

Reply:
Hey Greg, glad to have you back :)

Good catch! I actually had that in mind, thats why I left the source/destination/slotToclear slots as public setters.

It should be fixed now!